### ISSUE TITLE
Is this project still alive?

### ISSUE THREAD

**Issue by jhubert** [CenterForOpenScience/pydocx#246]
It looks like no pull-requests have been merged since February. Is this project still alive or can we do something else to help support? 😃

**Comment by jlward** [comment]
The project is still alive. We use it extensively. Right now, there are only two PRs with passing tests. I don't have a ton of time to review and merge. If you would like to review and merge issues, feel free to. If you need help with that feel free to let me or @kylegibson know and we can help as needed.

**Comment by jhubert** [comment]
@jlward Thanks for the quick response. I wasn't meaning to be offensive in case it came off that way.  

We'd hired botzill to work on edge cases we find, but he's moving on so we're checking to see what approach we should take next. I'm not as familiar with python so I can do my best to assist but it's likely we'll bring on someone else dedicated to this.

**Comment by jlward** [comment]
> Thanks for the quick response. I wasn't meaning to be offensive in case it came off that way.

No offense taken :)

> I'm not as familiar with python so I can do my best to assist but it's likely we'll bring on someone else dedicated to this.

Unfortunately, we don't really have the time to dedicate to fixing bugs that aren't affecting us directly (not very awesome of us, but there it is). I can try to be better about getting existing PRs reviewed and merged into master than I've currently been. If there are pressing PRs you need merged in, feel free to reach out to me and I can see what I can do to help it get merged in.

**Comment by jhubert** [comment]
Thanks Jason. We’re not blocked by anything because we run it all off of
our own repo but I do want to make sure we’re sharing back to the
community. No pressure on you guys, I know you’re busy and have your own
priorities.

Our instance of pydocx is heavily used to process thousands of documents.
If we can help in any way please don’t hesitate to ask.

**Comment by IuryAlves** [comment]
I would like to help, but i don`t have much free time either.
